### ISSUE TITLE
All Minting fees should be summarized at the first page

### ISSUE THREAD

**Issue by ramaruro** [keep-network/tbtc-dapp#235]
In the current version, you don't know how much it will cost you to mint tBTC, you find that out after the fact. That will drive users away.
All the Fees (both in BTC and in ETC) should be displayed at once, and I should see how much of the tBTC Ethereum Token I will get at the end. Seeing it all at once makes way more sense, whatever the fees are, to help decide whether you want to make the transaction or not
I also think the tBTC Minting Steps 1-5, although necessary from the protocol point of view, need to be summarized at the beginning. This is important for transparency and adoption.

As an example, the RenBTC bridge (very similar to Keep in concept) does that.
![RenBTC](https://user-images.githubusercontent.com/68167410/87234066-fddd4600-c392-11ea-8833-377a2ca6deba.JPG)

**Comment by mhluongo** [comment]
Agreed, I think being transparent up-front about cost is vital. @Gluzman @ironng could we start addressing this next week after the VIS update?

**Comment by Gluzman** [comment]
hi @ramaruro glad you brought this up. we've been thinking about this sort of improved UX for a bit now. we're in progress on developing an updated tBTC dapp experience and UI that addresses user expectations. you've hit the nail on the head.

here's a little rough wireframe preview of a layout we're considering that should address part of your issue. notice the sequence module, which is persistent and acts as a journey map for minting steps. not show, but also under consideration: full price/cost disclosure at every step. 

<img width="855" alt="Screen Shot 2020-07-13 at 11 53 57" src="https://user-images.githubusercontent.com/7621557/87325730-d657bc00-c4ff-11ea-8604-61dc81dc1bef.png">

hope this gives you something to look forward to, cheers!
